modules/ROOT/pages/database-internals/store-formats.adoc

```diff
@@ -0,0 +1,44 @@
+= Store formats
+:description: This page describes store formats in Neo4j.
+
+The store format is a core component of a database in Neo4j, describing how data is stored on disk and how to read it.
+
+== Store formats in Neo4j
+Neo4j has several defined store formats, each with a set of different features and limitations. The store format of a database is decided when the database is created, most commonly controlled by the `db.format` setting in the neo4j.conf file, or as an argument to the tool creating the database, e.g `import` or `copy` tools. The current available formats are
+
+* `aligned`
+* `standard`
+* `high_limit` label:enterprise-edition[]
+* `block` label:enterprise-edition[]
+
+== Properties of a store formats
+The store format is responsible for how data is written to disk and how to read it. Some key aspects that may differ between formats
+
+* The performance and resource consumption of read and write transactions. An operation may be faster on one store format than another.
+* The physical size on disk. Two databases with identical logical data on different format may have different size.
+* Performance in memory constrained environments. The same query on different formats may have different page cache hit ratio when the database does not fully fit in the page cache
+* The amount of data that can be stored. See xref:tools/neo4j-admin/neo4j-admin-store-info.adoc#neo4j-admin-store-entity-limits[entity limits] for limitations of the individual formats.
+* The order of query results when the order is not specified. Different store formats may traverse and return data in different order
+* Algorithms used by admin tools. Especially `check` and `import` commands. Performance and resource utilization may differ 
+
+
+== Changing store format
+Each format has a different layout of the data stored on disk. Changing the format is possible to do by re-writing all data in the new format, which only can be done when the database is offline. The `migrate` and `copy` tools provide this functionality.
+
+
+[NOTE]
+====
+Changing format of a database requires enough free disk space to hold both the current and the new database. It may also be a time consuming operation, depending on the size and complexity of the data in the database. The performance of migration is heavily dependent on the speed of the disk and the amount of available memory
```

Comment:
```suggestion
Changing the format of a database requires enough free disk space to hold both the current and the new database.
It may also be a time-consuming operation, depending on the size and complexity of the data in the database. The migration performance heavily depends on the speed of the disk and the amount of available memory.
```